dataprofiler/labelers/regex_model.py

```diff
@@ -250,11 +244,11 @@ def predict(
                         pred[indices[0] : indices[1], entity_id] = 1
             if verbose:
                 sys.stdout.flush()
-                sys.stdout.write("\rData Samples Processed: {:d}   ".format(i + 1))
+                sys.stdout.write(f"\rData Samples Processed: {i + 1:d}   ")
```

Comment:
This would be a functional change - the formatting syntax (ie. `:d` is the same in f-strings as it is with `format()`.

If y'all would like me to do that, happy to.

Reply:
Correct, `:d` is already formatting.

Let's leave it as is